specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/stable/2024-06-27/appcomplianceautomation.json

```diff
@@ -4221,10 +4221,7 @@
           "description": "List of resource ids to be evaluated",
           "items": {
             "type": "string"
-          },
```

Comment:
Can you remind me why some of these removals are ok/expected?

Reply:
x-ms-identifiers should be used for an array of objects: https://azure.github.io/autorest/extensions/#x-ms-identifiers so in this case it was applied to an array of strings